test/test_overrides.py

```diff
@@ -462,25 +462,69 @@ def test_user_implementation_raises(self):
             quux(t1)
 
 def generate_tensor_like_override_tests(cls):
-    def test_generator(func, override):
-        if torch._six.PY3:
-            args = inspect.getfullargspec(override)
+    from tools.autograd.gen_autograd import load_aten_declarations
```

Comment:
You can't do this. I need to read more to see why you want this file.

Reply:
I'm wondering if you could get this information from mypy annotations. (Making this test file py3 only, but that's fine.)

Reply:
There seem to be none yet (version 1.4.0):

```python
>>> import inspect
>>> import torch
>>> inspect.getfullargspec(torch.cat)
Traceback (most recent call last):
  File "/Users/hameerabbasi/miniconda3/envs/temp/lib/python3.7/inspect.py", line 1126, in getfullargspec
    sigcls=Signature)
  File "/Users/hameerabbasi/miniconda3/envs/temp/lib/python3.7/inspect.py", line 2288, in _signature_from_callable
    skip_bound_arg=skip_bound_arg)
  File "/Users/hameerabbasi/miniconda3/envs/temp/lib/python3.7/inspect.py", line 2112, in _signature_from_builtin
    raise ValueError("no signature found for builtin {!r}".format(func))
ValueError: no signature found for builtin <built-in method cat of type object at 0x115e93eb0>

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/hameerabbasi/miniconda3/envs/temp/lib/python3.7/inspect.py", line 1132, in getfullargspec
    raise TypeError('unsupported callable') from ex
TypeError: unsupported callable
>>> torch.cat.__annotations__
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'builtin_function_or_method' object has no attribute '__annotations__'
>>> def f(x: int) -> int:
...     pass
...
>>> f.__annotations__
{'x': <class 'int'>, 'return': <class 'int'>}
```